check_nextcloud.php

```diff
@@ -22,21 +23,75 @@ function convert_filesize($bytes, $decimals = 2) {
   return sprintf("%.{$decimals}f %s", $bytes / pow(1024, $factor), @$size[$factor]);
 }
 
+function check_range($pattern,$value) {
```

Comment:
Can you elaborate (add inline comments), what exactly this function and the if-clauses/preg_matches do? Please also add a DocBlock describing input parameters and return value.